lambda/create/main.go

```diff
@@ -77,10 +85,10 @@ func (l *Lambda) HandleEvent(ctx context.Context, req events.APIGatewayProxyRequ
 	}
 
 	// validation
-	errors := Validate(input)
-	if len(errors) > 0 {
+	errs := Validate(input)
+	if len(errs) > 0 {
```

Comment:
these can be written like `if errs := Validate(input); len(errs) > 0 {` or `if err := l.store.Put(ctx, data); err != nil {` instead, which I think looks cleaner and scopes the variable to that block